src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/git/dialog/GitReviewPresenter.java

```diff
@@ -486,6 +491,23 @@ public void onValueChange(ValueChangeEvent<Boolean> event)
                      updateDiff(false);
                }
             });
+      
+      view_.getIgnoreWhitespaceCheckBox().setValue(
+            uiPrefs_.gitDiffIgnoreWhitespace().getGlobalValue());
+      
+      view_.getIgnoreWhitespaceCheckBox().addValueChangeHandler(
+            new ValueChangeHandler<Boolean>()
+            {
+               @Override
+               public void onValueChange(ValueChangeEvent<Boolean> event)
+               {
+                  boolean value = event.getValue();
+                  uiPrefs_.gitDiffIgnoreWhitespace().setGlobalValue(value);
+                  uiPrefs_.writeUIPrefs();
```

Comment:
It's a little heavyweight to push all the UI prefs to the server every time the box is checked. Would it be difficult to do this on commit instead?